src/validations/actionSchema.ts

```diff
@@ -60,14 +60,12 @@ export const extendLifecycleIdSchema = Yup.object().shape({
           year: Number(newExpirationYear) || 0
         });
 
-        const oldExpiration = DateTime.fromISO(currentExpiresAt) || 0;
-        if (
-          newExpiration.isValid &&
-          newExpiration.toMillis() > oldExpiration.toMillis()
-        ) {
-          return true;
+        const oldExpiration = DateTime.fromISO(currentExpiresAt);
```

Comment:
This looks great, can you add a comment explaining that were checking the old expiration validity to handle cases where the old date is invalid (usually due to bad SharePoint data migration)

Reply:
Done